### PR TITLE
PLANET-7255: Move Gallery block to master theme

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-7255

## Tests
On [test instance](https://www-dev.greenpeace.org/test-umbriel/) or locally:
- Enable and disable _Planet4 > Features > Planet 4 Blocks_ and verify Gallery scripts are coming from master-theme (enabled) or gutenberg-block (disabled) and that the block behaviour (frontend, editor, existing and new block) stays the same.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [umbriel](https://www-dev.greenpeace.org/test-umbriel) | [admin](https://www-dev.greenpeace.org/test-umbriel/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-umbriel/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-umbriel) | [composer-local.json](https://github.com/greenpeace/planet4-test-umbriel/blob/main/composer-local.json)

:watch: 2023.12.19 09:53:37

File: assets/src/blocks/Gallery/useGalleryImages.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const args = {
+      image_size: imageSize,
+      multiple_image,
+      gallery_block_focus_points,
+    };
+
+    try {
+      // eslint-disable-next-line no-shadow
+      const images = baseUrl ?
+        await fetchJson(`${baseUrl}/wp-json/${addQueryArgs('planet4/v1/get-gallery-images', args)}`) :
```
Comment (Osong-Michael) [inline assets/src/blocks/Gallery/useGalleryImages.js:30]: I get this error below in the console, shows you are not registering the route in the master-theme.

Screenshot 2023-12-20 at 10 47 15

**Review by Osong-Michael — COMMENTED** [review]
The images don't show up in the editor, just the sidebar.

scrnli_20_12_2023_10-57-24

**Review by Osong-Michael — APPROVED** [review]
👍 code looks good to me

PS: Just following @mleray suggestion, we could replace `<Fragment></Fragment>` with just empty `<></>`

**Pull request merged**